### PR TITLE
Bug fix and refactoring - the navigation buttons should work correctly again

### PR DESCRIPTION
**Pull request by SantiagoDePolonia**

### CODE REVIEW

File: assets/js/Navigator.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ export default function Navigator({
     const isPrevChapterOrBookAvailable =
         isPrevChapterAvailable() || isPrevBookAvailable();
     return (
-        <header className="container sticky-top pt-2 pb-2">
+        <header className="container sticky-top pt-2 pb-2 user-select-none">
```
Comment (clash82) [inline assets/js/Navigator.js:32]: What's this class? Where it comes from?

**Review by clash82 — CHANGES_REQUESTED** [review]
Refactoring is really great here 🏆 One minor comment was added, review please.

_If you want to push something to this branch then pull it first - branch was rebased on top of latest `master`._

**Comment by clash82** [comment]
@SantiagoDePolonia could you also check SonarCloud report? There are 9 new issues found.

**Comment by SantiagoDePolonia** [comment]
Those SonarCloud issues are not new. All of them was present before. However I fixed one of them. Other are not very important and require some redundant work at this stage of the project. (Especially if you consider introducing typescript propTypes would be redundant. )

**Comment by clash82** [comment]
> Those SonarCloud issues are not new. All of them was present before. However I fixed one of them. Other are not very important and require some redundant work at this stage of the project. (Especially if you consider introducing typescript propTypes would be redundant. )

Yeah, we can skip SonarCloud recommendations for now. Could you also check my comment here https://github.com/rBiblia/rbiblia-web/pull/17#discussion_r1554714121

File: assets/js/Navigator.js (hunk 1 above)
Comment (SantiagoDePolonia) [reply assets/js/Navigator.js:32]: It comes from bootstrap.

Comment (SantiagoDePolonia) [reply assets/js/Navigator.js:32]: It is css user-select:none; property

Comment (clash82) [reply assets/js/Navigator.js:32]: Nice, I didn't know that.

**Review by clash82 — APPROVED** [review]

**Pull request merged**